package.json

```diff
@@ -24,30 +24,30 @@
   ],
   "license": "MIT",
   "dependencies": {
-    "bluebird": "^3.5.1",
-    "chalk": "^2.4.1",
-    "compression": "^1.7.3",
-    "connect": "^3.6.6",
-    "mime": "^2.4.0",
-    "morgan": "^1.9.0",
-    "opn": "^5.3.0",
-    "serve-static": "^1.13.2"
+    "bluebird": "^3.5.5",
+    "chalk": "^2.4.2",
+    "compression": "^1.7.4",
+    "connect": "^3.7.0",
+    "mime": "^2.4.3",
+    "morgan": "^1.9.1",
+    "open": "^6.3.0",
+    "serve-static": "^1.14.1"
   },
   "devDependencies": {
     "chai": "^4.2.0",
     "chai-as-promised": "^7.1.1",
-    "eslint": "^5.7.0",
+    "eslint": "^5.16.0",
     "eslint-config-hexo": "^3.0.0",
-    "hexo": "^3.7.1",
-    "hexo-fs": "^1.0.0",
-    "istanbul": "^0.4.1",
-    "mocha": "^6.0.2",
-    "sinon": "^7.0.0",
-    "supertest": "^1.2.0",
+    "hexo": "^3.8.0",
+    "hexo-fs": "^1.0.2",
+    "istanbul": "^0.4.5",
+    "mocha": "^6.1.4",
+    "sinon": "^7.3.2",
+    "supertest": "^4.0.2",
     "supertest-promised": "^1.0.0",
     "uuid": "^3.3.2"
   },
   "engines": {
-    "node": ">=6.9.0"
+    "node": ">=8"
```

Comment:
IMO `>=8.6.0` is better. It's the same with https://github.com/hexojs/hexo/pull/3598

Reply:
Done